packages/features.lua

```diff
@@ -1,7 +1,11 @@
+-- OpenType features for SILE.
```

Comment:
No inline copyright headers please, the project level notice covers this.

Reply:
Happy to remove it, but I'd like this to change:

https://github.com/sile-typesetter/sile/blob/3d0c0a3a22a758029e252e2e2da8f26af7f7c064/LICENSE#L3

On FontForge we solve this by having an `AUTHORS` file, and in the `LICENSE`, having:

## Contributors

See [`AUTHORS`](https://github.com/fontforge/fontforge/blob/master/AUTHORS).

And then in `AUTHORS`, we recognize our project's founder as being most significant by putting him on top, and then we list all committers alphabetically. Perhaps by number of commits would be more fair, I hadn't considered it deeply before this. Remembering, of course, that our project is much older, and went through several CVS, so in our case, AUTHORS doesn't equal one to one with commits, as sometimes George just thanked someone in a commit message and put them in authors, while still marking the code as his in Author field.

Reply:
Bumping the date in the license and adding an MIT appropriate authors file would be welcome (the authors file if any should include a make target to update it before releases by scanning Git history), but that's not part of this PR. PRs need to stick to discrete topics.